README.md

```diff
@@ -1 +1,24 @@
-# jsonforms-vue-seed
\ No newline at end of file
+# jsonforms-vue-seed
+
+## Project setup
+```
+yarn install
```

Comment:
I would prefer to use `npm ` over `yarn` for simple projects like this. Therefore please also remove the `yarn.lock` and update the `package-lock.json`